### PR TITLE
Update styling and placement for sync warning banner

### PR DESCRIPTION
**Pull request by austin-martinez**
Related to https://github.com/department-of-veterans-affairs/sunsets-team/issues/257

screen shot 2017-02-01 at 5 34 13 pm
screen shot 2017-02-01 at 5 34 42 pm
screen shot 2017-02-01 at 5 38 21 pm

### CODE REVIEW

File: src/js/disability-benefits/components/ClaimSyncWarning.jsx [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
 
-export default function ClaimSyncWarning() {
+export default function ClaimSyncWarning({ olderVersion }) {
+  let additionalText;
+  if (olderVersion) {
+    additionalText = ' This is an older version of your claim and may be outdated.';
+  }
+
+  function handleClick(e) {
+    e.preventDefault();
```
Comment (jbalboni) [inline src/js/disability-benefits/components/ClaimSyncWarning.jsx]: I don't think this necessary

**Review by jbalboni — APPROVED** [review]
LGTM

File: src/js/disability-benefits/components/ClaimSyncWarning.jsx (hunk 1 above)
Comment (austin-martinez) [inline src/js/disability-benefits/components/ClaimSyncWarning.jsx]: This https://facebook.github.io/react/docs/handling-events.html made it seem like it was

Comment (jbalboni) [inline src/js/disability-benefits/components/ClaimSyncWarning.jsx]: You're refreshing the page, so you should be skipping the default behavior anyway, I mean. Not a big deal either way, I don't think.

**Pull request merged**